WordPress/Classes/Utility/WPUserAgent.h

```diff
@@ -6,6 +6,8 @@
  */
 @interface WPUserAgent : NSObject
 
++ (instancetype) sharedInstance;
```

Comment:
We chatted about this earlier, but I'd cache just the default user agent instead of having a shared instance.